lib/AppInfo/Capabilities.php

```diff
@@ -9,21 +9,23 @@
 
 namespace OCA\Notes\AppInfo;
 
+use OCA\Notes\Service\NoteUtil;
 use OCP\App\IAppManager;
 use OCP\Capabilities\ICapability;
+use OCP\IUserSession;
 
 class Capabilities implements ICapability {
-	private IAppManager $appManager;
-
-	public function __construct(IAppManager $appManager) {
-		$this->appManager = $appManager;
+	public function __construct(private IAppManager $appManager,
+								private NoteUtil $noteUtil,
+								private IUserSession $userSession) {
 	}
 
 	public function getCapabilities(): array {
 		return [
 			Application::APP_ID => [
 				'api_version' => Application::$API_VERSIONS,
 				'version' => $this->appManager->getAppVersion(Application::APP_ID),
+				'notes_path' => $this->noteUtil->getOrCreateNotesFolder($this->userSession->getUser()->getUID())->getInternalPath()
```

Comment:
Maybe this?

```suggestion
				'notes_path' => $this->noteUtil->getOrCreateNotesFolder($this->userSession->getUser()?->getUID())->getInternalPath()
```

Reply:
This also gives /files/Notizen" , but not "/Notizen"